src/cleanlab_codex/internal/validator.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any
+
+from cleanlab_codex.utils.errors import MissingDependencyError
+
+try:
+    from cleanlab_tlm.utils.rag import Eval, TrustworthyRAGScore, get_default_evals
```

Comment:
Related to the comment above, I'd suggest just switching this to a regular import and making this a required dependency.